docs/systems/templates.md

```diff
@@ -0,0 +1,13 @@
+# System Templates
+
+When creating a new system in the System Designer, users have the option of starting with either a blank system or a system template.
+
+<img src="/img/system_template.png" class="right border" width="300" />
+
+Using a system template launches a nio system with pre-configured functionality and development notes across instances, services, and blocks.
+
+System templates are great for quickly learning new concepts and skills (we’ve built several workshop templates) and/or jump starting your next production system.
+
+Make sure to check out the available templates next time you launch a new system! We’re always adding to the library.
```

Comment:
can we add a link to a list of the available templates somewhere?

Reply:
Good idea—just found the list.